src/e2e-test/java/io/cdap/plugin/dataplex/stepsdesign/DataPlexBase.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.dataplex.stepsdesign;
+
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.plugin.dataplex.actions.DataPlexActions;
+import io.cdap.plugin.dataplex.locators.DataPlexLocators;
+import io.cucumber.java.en.Then;
+
+/**
+ * PubSub Source Plugin related step design.
+ */
+
+public class DataPlexBase implements CdfHelper {
```

Comment:
Use proper steps from e2e framework PipelineSteps.java.
Most of the step definitions from this file required.